wildfly-builder-image/image.yaml

```diff
@@ -52,8 +54,9 @@ modules:
           - name: dynamic-resources
           - name: jboss.container.maven.default
           - name: jboss.container.maven.s2i
-          - name: jboss.container.wildfly.s2i.bash
-
+          - name: jboss.container.wildfly.s2i-wildfly.bash
+          # Remove this dependency to not support env variable based provisioning and legacy workflow
+          - name: jboss.container.wildfly.s2i.legacy
```

Comment:
is there a notion of "deprecated" in cekit modules?

Reply:
Looking at modules metadata I didn't find a way to tag a module as deprecated.